pkg/metrics/metrics.go

```diff
@@ -133,6 +163,11 @@ func (m *Metrics) Describe(ch chan<- *prometheus.Desc) {
 	ch <- deviceFreeInBytes
 	ch <- deviceUsedInBytes
 	ch <- brickCount
+	ch <- staleCount
```

Comment:
It would be nice to have a comment here delineating the "sub sections" of items.

Reply:
Added comment